src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer.mdx

```diff
@@ -27,7 +27,12 @@ import kubernetesClusterExplorerPickerRefresh from 'images/kubernetes_screenshot
 
 import kubernetesEvents from 'images/kubernetes_screenshot-full_events.gif'
 
-New Relic's Kubernetes cluster explorer uses the data collected by the [Kubernetes integration](/docs/integrations/kubernetes-integration/get-started/introduction-kubernetes-integration) to show the status of your cluster, from the control plane to nodes and pods. You can find out about the health of each entity, explore logs, and see how your apps are performing. With the [Events integration](/docs/integrations/kubernetes-integration/kubernetes-events/install-kubernetes-events-integration), everything that happens in your cluster becomes visible, and logs brought in using the [logs plugin](/docs/logs/enable-logs/enable-logs/kubernetes-plugin-logs) are also available.
+
+Kubernetes cluster explorer of New Relic uses the data collected by the [Kubernetes integration](/docs/integrations/kubernetes-integration/get-started/introduction-kubernetes-integration) to show the status of your cluster, from the control plane to nodes and pods. 
```

Comment:
I can't add this change because we don't make New Relic or its platform, capabilities, and so on possessive using apostrophes. See [Product and capabilities: capitalization, trademarks, and usage](https://docs.newrelic.com/docs/style-guide/capitalization/product-capability-feature-usage/).